### ISSUE TITLE
I have a problem installing all flutter versions and channels 

### ISSUE THREAD

**Issue by techwithsam** [leoafarias/sidekick#105]
I have a problem installing all flutter version and channel including changing settings, while installation process done correctly 
```
Internal Error: could not save FVM settings
```
![Screenshot 2021-06-23 090222](https://user-images.githubusercontent.com/42917957/123059767-02630900-d402-11eb-92cc-050f3aaa656b.png)

**Comment by aguilaair** [comment]
Do you have FVM installed?

**Comment by techwithsam** [comment]
Yes, I installed via power shell (admin privileges) -- choco install fvm

![Screenshot 2021-06-23 122950](https://user-images.githubusercontent.com/42917957/123089491-ea4db280-d41e-11eb-8bb5-cc39ce8b903c.png)

**Comment by leoafarias** [comment]
@techwithsam please run a “fvm doctor” and post the output

**Comment by techwithsam** [comment]
![Screenshot 2021-06-23 124431](https://user-images.githubusercontent.com/42917957/123091348-2550e580-d421-11eb-8575-9604d15665ee.png)

**Comment by leoafarias** [comment]
@techwithsam Could you do an `fvm install 2.2.2 --verbose` and provide the output?

**Comment by techwithsam** [comment]
![Screenshot 2021-06-24 001833](https://user-images.githubusercontent.com/42917957/123179662-c5d6f200-d481-11eb-8d7d-aca458439f15.png)
